src/xrp/xrpEngine.ts

```diff
@@ -293,14 +288,77 @@ export class XrpEngine extends CurrencyEngine<RippleTools> {
           this.updateBalance(edgeToken.currencyCode, assetAmount)
         }
       })
+
+      // If get here, we've checked balances for all possible tokens the user
+      // could have enabled. Mark all assets as checked
+      this.enabledTokens.forEach(tokenCurrencyCode => {
+        this.tokenCheckBalanceStatus[tokenCurrencyCode] = 1
+      })
+      this.updateOnAddressesChecked()
+
+      if (this.enabledTokens.length > 1) {
+        // Check for unactivated tokens
+        const acctLinesResponse = await this.tools.rippleApi.request({
+          command: 'account_lines',
+          account: address
+        })
+
+        this.enabledTokens.forEach(tokenCurrencyCode => {
+          const match = acctLinesResponse.result.lines.find(line => {
+            const { account: issuer, currency } = line
+            const lineTokenId = makeTokenId({ currency, issuer })
+            const edgeToken = this.allTokensMap[lineTokenId]
+            if (
+              edgeToken != null &&
+              tokenCurrencyCode === edgeToken.currencyCode
+            ) {
+              return true
+            }
+            return false
```

Comment:
You could just `return edgeToken != null && tokenCurrencyCode === edgeToken.currencyCode`.

Reply:
👍 I had those lines in for breakpoint testing. Unneeded now.